server2/Arcadia.Assistant/Arcadia.Assistant.Notifications.Contracts/INotifications.cs

```diff
@@ -0,0 +1,23 @@
+namespace Arcadia.Assistant.Notifications.Contracts
```

Comment:
assembly V2 and everything else. That also applies to all other services

Reply:
Fixed.